Libs/MRML/Core/vtkMRMLVolumeNode.cxx

```diff
@@ -1050,7 +1050,7 @@ void vtkMRMLVolumeNode::ApplyNonLinearTransform(vtkAbstractTransform* transform)
     }
 
   reslice->SetInputConnection(this->ImageDataConnection);
-  reslice->SetInterpolationModeToLinear();
+  reslice->SetInterpolationMode(this->GetResliceInterpolationMode());
```

Comment:
@pieper I've tried to do that over here.

Reply:
Ah, I see, I thought you'd only added the accessors.